### PR TITLE
Enable Container Networking

### PR DESCRIPTION
**Pull request by spatchineelam**
Local Networking is enabled

### CODE REVIEW

File: pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
++        domain|dns)
++          REDO_DNS='yes'
++          ;;
++      esac
++    done
++    # save new config info:
++    mv -f ${CFG}.new $CFG
++    # make only necessary changes, as per config comparison:
++    if [ -n "$REDO_NET" ] ; then
++      ip addr flush dev $interface
```
Comment (eriknordmark) [inline pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch]: According to the udhcp documentation the IP address will never change when you get a renew. So these two lines are not needed.

File: pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
++      ip addr add ${ip}/${mask} dev $interface
++      [ -n "$router" ] && ip route add default via ${router%% *} dev $interface
```
Comment (eriknordmark) [inline pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch]: Where do you delete the old default route?

File: pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch (hunk 1 above)
Comment (eriknordmark) [reply pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch]: Ah, but the subnet mask can change. I guess that means you need to delete and re-add the ip addr.

File: pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
++    ;;
++  renew)
++    echo "udhcpc op renew interface ${interface}"
++    # compare new vs. previous config info:
++    set > ${CFG}.new
++    for i in $(diff -U1 $CFG ${CFG}.new | grep -E ^[+-] \
++                                        | tail +3 \
++                                        | awk -F[+-=] '{print $2}') ; do
++      case "$i" in
++        ip|mask|router)
```
Comment (eriknordmark) [inline pkg/rkt-stage1/0006-Enable-local-networking-in-containers.patch]: You should never see "ip" here.

**Comment by rvs** [comment]
@spatchineelam it makes sense for now as a stopgap, but please create an extra commit and PR identifying where you took the source for the dhcp script from (I suspect it is Alpine)

This is required for IP tracking and is kind of a pretty serious deal whenever we import existing code into our tree

**Comment by spatchineelam** [comment]
Opened https://github.com/lf-edge/eve/pull/247 for identifying the source of dhcpc script

**Pull request merged**